datagovharvester/extract/dcatus.py

```diff
@@ -0,0 +1,57 @@
+from .utils import download_catalog
```

Comment:
The pattern below of `from datagovharvester.utils.xxx` makes sense to me. Why did you want to do something differently here?

Reply:
Most likely for convenience but with autocomplete that's not really an excuse haha. good find. i've updated my imports to be absolute rather than relative.

Reply:
I refactored a bit so i'm not importing the same thing(s). [similar lines in the same module](https://github.com/GSA/datagov-harvesting-logic/blob/ba005adfef0b0f1c2d94fe630bf509d637bf57a1/datagovharvester/extract/dcatus.py#L2-L3)

Reply:
Sorry, I don't think I was very clear. Why do we have `utils` here and a `utils` dir with sub `utils`? Do you think it would make more sense to group all of them in the same place (honest question, do you think it is worth it to separate them)?

Reply:
My thinking is `datagovharvester.utils` are modules applicable anywhere. For example, the `datagovharvester.utils.s3_utilities.upload_to_S3` function will be, or is currently, used in the `extract`, `transform`, and `load` main functions found in their respective `__init__.py` modules ( currently only `extract` ). `datagovharvester.extract.utils` are functions which are primarily applicable to `extract`. All the utils could be placed in `datagovharvester.utils`. It would simplify things and having all of them in one place could make navigation and the general structure easier to work with.

Reply:
Hmm, sounds like both approaches have merit. I bringing it up since it would set a precedent for us: unified or separate `utils`. What does the rest of the @GSA/data-gov-team think?

To be clear @rshewitt, I'm happy to do the refactor. I'm not suggesting you do the work, more how should we approach this now and in the future.